doc/source/release_notes.rst

```diff
@@ -1,6 +1,10 @@
 Release notes
 =============
 
+v0.3.4
+------
+- API: Make ``frame_shape`` match frames' ``shape`` in ``SpeStack`` (PR 241)
```

Comment:
An alternative would be to make a new property (image_size?) and have frame_size issue a DeprecationWarning.

On Sep 2, 2016, at 4:44 AM, Thomas A Caswell <notifications@github.com<mailto:notifications@github.com>> wrote:


In doc/source/release_notes.rst<https://github.com/soft-matter/pims/pull/241#discussion_r77333191>:

> @@ -1,6 +1,10 @@
>  Release notes
>  =============
>
> +v0.3.4
> +------
> +- API: Make ``frame_shape`` match frames' ``shape`` in ``SpeStack`` (PR 241)


Can you also say that the order has flipped? The target audience for this text is someone who updated and now their code is broken so they may not be in the mood to dig through code / PRs ? .

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/soft-matter/pims/pull/241/files/65d947fe9707ea3ddad076740eafa4fefb320c2e#r77333191>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AC2NbXYTyH7HVbmlxuQ0mUrbK8eD2x06ks5qmAwHgaJpZM4JymHO>.